README.md

```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
 ./react-bootstrap/run.sh -l
 ```
 
+## Bootstrap options
+
+You can choose between two creation types: Clean and Customized
+
+- ### Clean
+
+Create a clean project with the most used libraries of React
```

Comment:
`Create a clean project using crate-react-app as baseline and with some small addition to its configuration`